tests/_utils.rs

```diff
@@ -73,13 +73,14 @@ impl Default for SampleWorkspace {
     /// Set up a temporary directory for tests.
     fn default() -> Self {
         let temp = Temp::new_dir().expect("create working directory for sample workspace");
+        let long_str = "a".repeat(1000);
```

Comment:
Instead of declaring this variable, you can change the type of the tree instead:

```rust
MergeableFileSystemTree::<&str, String>::from(dir! {
```